app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,12 @@
 <resources>
     <string name="app_name">DemoApplication</string>
+    <string name="title_activity_main">MainActivity</string>
+    <!-- Strings used for fragments for navigation -->
+    <string name="first_fragment_label">First Fragment</string>
+    <string name="second_fragment_label">Second Fragment</string>
+    <string name="next">Next</string>
+    <string name="previous">Previous</string>
+
+    <string name="hello_first_fragment">Hello first fragment</string>
+    <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/luudan0079/DemoRepo/blob/1a7e14139b335ac24c031a60062d1f5dbd39a0e9/app/src/main/res/values/strings.xml#L11"></span>The resource <code>R.string.hello_second_fragment</code> appears to be unused</td>
    </tr>
  </tbody>
</table>